src/components/MediaCard/component.js

```diff
@@ -16,16 +16,19 @@ export const MediaCard = ({
   ratingTwenty,
   ratingSystem,
   style,
+  loading,
 }) => {
   const onPress = () => {
-    navigate('MediaPages', {
-      mediaId: mediaData.id,
-      mediaType: mediaData.type,
-    });
+    if (mediaData.id && mediaData.type) {
+      navigate('MediaPages', {
+        mediaId: mediaData.id,
+        mediaType: mediaData.type,
+      });
+    }
   };
 
   return (
-    <TouchableOpacity onPress={onPress}>
+    <TouchableOpacity onPress={!loading && onPress}>
```

Comment:
Can we use `disabled` prop instead of this? like `<TouchableOpacity onPress={onPress} disabled={loading}>`